repo-config.csv

```diff
@@ -1,5 +1,5 @@
 Repository's Location,Branch,File formats,Ignore Glob List,Ignore standalone config
 https://github.com/reposense/testrepo-Alpha.git,master,,,
-https://github.com/reposense/testrepo-Beta.git,master,java;adocs,docs**,yes
-https://github.com/reposense/testrepo-Delta.git,master,html;css,,
-https://github.com/reposense/RepoSense.git,master,,,
+https://github.com/reposense/testrepo-Beta.git,master,fxml,docs**,yes
+https://github.com/reposense/testrepo-Delta.git,master,,,
+https://github.com/reposense/testrepo-Delta.git,nonExistentBranch,,,
```

Comment:
Please add back the RepoSense as we want to continuously visualize a `live` repo.
Others can all follow the properties of functional config.